stories/core/Table.stories.tsx

```diff
@@ -1324,3 +1325,100 @@ Full.args = {
 
 export const Condensed: Story<TableProps> = Basic.bind({});
 Condensed.args = { density: 'condensed' };
+
+export const Editable: Story<TableProps> = (args) => {
+  const { ...rest } = args;
+
+  type TableStoryDataType = {
+    name: string;
+    description: string;
+  };
+
+  const [data, setData] = React.useState(() => [
+    { name: 'Name1', description: 'Description1' },
+    { name: 'Name2', description: 'Description2' },
+    { name: 'Name3', description: 'Fetching...' },
+  ]);
+
+  const isRowDisabled = useCallback((rowData: TableStoryDataType) => {
+    return rowData.name === 'Name2';
+  }, []);
+
+  const columns = React.useMemo(
+    (): Column<TableStoryDataType>[] => [
+      {
+        Header: 'Table',
+        columns: [
+          {
+            id: 'name',
+            Header: 'Name',
+            accessor: 'name',
+            isCellEditable: () => true,
+            Filter: tableFilters.TextFilter(),
+          },
+          {
+            id: 'description',
+            Header: 'Description',
+            accessor: 'description',
+            isCellEditable: (rowData) => rowData.description !== 'Fetching...',
+            Filter: tableFilters.TextFilter(),
+          },
+        ],
+      },
+    ],
+    [],
+  );
+
+  const onCellEdit = useCallback(
```

Comment:
Not sure if you plan to keep this approach, but if you do, then you should add an example like this callback in the JSDoc comment of `onCellEdit` prop. Because it's not very obvious that the user needs to update the data on their end.

Reply:
Added a comment on `onCellEdit` prop.